docs/content/references/config/auth/ldap.md

```diff
@@ -0,0 +1,24 @@
+# LDAP
+
+HedgeDoc can use one or multiple LDAP servers to authenticate users. To do this,
+you first need to tell HedgeDoc the names of servers you want to use (`HD_AUTH_LDAPS`),
```

Comment:
Whoops, potential for confusion: This variable is called `LDAPS` meaning "multiple LDAPs", but "LDAPS" also means "LDAP Secure".

Reply:
Maybe `HD_AUTH_LDAP_SERVERS`?